lib/subscriber_list_mover.rb

```diff
@@ -47,5 +50,38 @@ def call
     end
 
     puts "#{subscribers.count} active subscribers moved from #{from_slug} to #{to_slug}"
+
+    if send_email
```

Comment:
This won't work at the moment as it's using the subscriber list for the old slug (which at this point has no subscribers as we've moved them all). I guess our options are:

1. Send the email first (risky if we send the email and then the move fails)
2. Keep track of each successful move and pass that data to the `email_change_to_subscribers` method (although I'm not sure exactly what data it's expecting).

Reply:
Thanks @vanitabarrett - I'll rework this for option 2 I think, if I can.

Reply:
I've gone with option 1 as the BulkEmailService/BulkEmailBuilder only supports sending emails to a SubscriberList and includes a SQL query. I don't think I could build a transient SubscriberList to keep track and then email the users of. See https://github.com/alphagov/email-alert-api/pull/1216/commits/18b301b04ddd5adc4ca8360e833c91754117356c